docs/code_contribution_guidelines.md

```diff
@@ -307,9 +308,32 @@ new breaking change to a module's API is introduced:
 - Repeat the process for any other modules the require a new major as a result
   of consuming the new major(s).
 
+<a name="#HandlingBackports" />
+
+### 4.6 Handling Backports
+
+Backported pull requests are required to reference the pull request being 
+applied to a prior release. The pplied pull request number is expected to be
```

Comment:
```suggestion
applied to a prior release. The applied pull request number is expected to be
```

Reply:
Corrected and updated the section, please take another look.